.coderabbit.yaml

```
@@ -47,182 +49,343 @@
+  dotenvLint:
+    enabled: true
+  shellcheck:
+    enabled: true
+  
+  # GitHub integration - ENHANCED
+  github-checks:
+    enabled: true
+    timeout_ms: 180000  # Increased for complex pipelines
+    wait_for_completion: true
+    analyze_failures: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)